auth/google.js

```diff
@@ -17,19 +13,21 @@ const passportConfig = {
 
 passport.use(new passportGoogle.OAuth2Strategy(passportConfig, async function (request, accessToken, refreshToken, profile, done) {
     try {
-        // the profile contains information being sent from google about the user.
         const { id, displayName } = profile;
         const emails = profile.emails.map(email => {
-            return {cust_id: id, email: email.value}
+            let idPropName = request.admin ? 'admin_id' : 'cust_id';
+            return { [idPropName]: id, email: email.value}
         });
+        const group = request.admin ? 'admins' : 'customers';
```

Comment:
Good idea to implement this way! 
what do you think about just adding a flag to the request body from the front end to do this?